src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+using Microsoft.WindowsAzure.MobileServices;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        // POCO output binding 
+        //   The binding inserts the newly created item into the Easy Table when the 
+        //   method successfully exits.         
+        public static void InsertItem([EasyTable] out Item newItem)
+        {
+            newItem = new Item()
+            {
+                Id = Guid.NewGuid().ToString(),
+                Text = new Random().Next().ToString()
+            };
+        }
+
+        // Query input binding 
+        //   The binding creates a strongly-typed query against the Item table. 
+        //   The binding does not do anything with the results when the function exits.  
+        //  This example takes the results of the query and puts them in a queue for further 
+        //  processing. 
+        public static async Task EnqueueItemToProcess(
```

Comment:
It actually wasn't intentional but I think it's okay. I actually resorted to excluding all the sample files, and 'Use' calls to test so I didn't have to set everything else up that was required. Maybe there's a way to make this work selectively in the future. But I think for now it's okay -- the code files themselves serve as sample documentation and I can jump in and get them running with a few lines when I want to run through something.